Tasks/Common/func-tools-common/functoolsutility.ts

```diff
@@ -0,0 +1,79 @@
+"use strict";
```

Comment:
Is there any other task using this common module? If not you can keep it within the task itself